Libs/MRML/Widgets/qMRMLSliceInformationWidget.cxx

```diff
@@ -246,27 +238,15 @@ void qMRMLSliceInformationWidget::setWidgetVisible(bool visible)
 //---------------------------------------------------------------------------
 void qMRMLSliceInformationWidget::setLightboxLayoutRows(int rowCount)
 {
-  Q_D(qMRMLSliceInformationWidget);
-
-  if (!d->MRMLSliceNode)
-  {
-    return;
-  }
-
-  d->MRMLSliceNode->SetLayoutGridRows(rowCount);
+  Q_UNUSED(rowCount);
```

Comment:
I marked `setLightboxLayoutRows/setLightboxLayoutColumns` those as deprecated for now.

Reply:
Is the API used outside?
If not, I would suggest removing the lightbox alltogether from the Qt level (or be consistent with the deprecation with the qMRMLSliceControllerWidget).